src/components/molecules/EventPaymentButton/EventPaymentButton.tsx

```diff
@@ -1,89 +0,0 @@
-import React from "react";
-import "firebase/functions";
-import { Link } from "react-router-dom";
-
-import { VenueEvent } from "types/venues";
-
-import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
-import { isUserAMember } from "utils/isUserAMember";
-import { isEventStartingSoon } from "utils/event";
-import { WithId } from "utils/id";
-import { venueEntranceUrl } from "utils/url";
-import {
-  currentVenueSelectorData,
-  userPurchaseHistorySelector,
-} from "utils/selectors";
```

Comment:
I don't see any changes to `utils/selectors` to remove `userPurchaseHistorySelector` in this PR. I'm guessing this can probably be removed now?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40